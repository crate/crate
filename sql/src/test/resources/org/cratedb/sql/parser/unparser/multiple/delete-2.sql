DELETE FROM t1;
DELETE FROM t2 WHERE x > 10;
DELETE FROM t3