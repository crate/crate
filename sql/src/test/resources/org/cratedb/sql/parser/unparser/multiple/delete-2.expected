[0]: DELETE FROM t1;
[1]: DELETE FROM t2 WHERE x > 10;
[2]: DELETE FROM t3;